content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -217,6 +217,11 @@ The completion script itself is generated by kubectl, so you typically just need
 Common examples are provided here. For more details, consult `kubectl completion -h`.
 
 ### On Linux, using bash
+On CentOS Linux, you will need to install bash-completion support, bash-completion not install in default.
+
```

Comment:
you may need to install the `bash-completion` package which is not installed by default.